### ISSUE TITLE
Unable to retrieve settings from secrets.yml

### ISSUE THREAD

**Issue by govorov** [telegram-bot-rb/telegram-bot#26]
Hi there!
I have a new empty rails application with `telegram-bot` installed. And the problem is `telegram_config` is empty despite the options in `secrets.yml`

There's my secrets.yml:
```
development:
  secret_key_base: <secret key base>
  telegram:
    bots:
      main: <here goes my bot api key>
```

And in rails console I get this:
```
#secrets are here, everything is fine
[3] pry(main)> Rails.application.secrets[:telegram]
=> {:bots=>{:main=>"378827954:AAFl-3Zk0xHF3McEmf2sOYnteZShxxZAUGY"}}
[4] pry(main)> 

#but telegram config is empty
[1] pry(main)> Telegram.bots
=> {}
```

I've tried to figure out what's happening and the reason is string hash keys there:
https://github.com/telegram-bot-rb/telegram-bot/blob/0c9a4a5fa00c17d4af888341c26e60396a045099/lib/telegram/bot/config_methods.rb#L52

Thank you!

**Comment by printercu** [comment]
Which rails version do you have? Maybe string keys have been changed to symbols. I've got 5.0.1, and all nested hashes have string keys.

**Comment by printercu** [comment]
Faced similar problem in new app with 5.1.0. Nested hashes has symbol keys without indifferent access. Will check rails issues for more info.

**Comment by govorov** [comment]
Sorry for so late response.
I'm using rails 5.1.0.rc1.

**Comment by printercu** [comment]
Found this pr: https://github.com/rails/rails/pull/26929. I wan't to wait for the response, may it'll be changed to indifferent access till release. For now you can add `Telegram.bots_config.deep_stringify_keys!` to initializers or before first access to bot.

**Comment by printercu** [comment]
Released fix in 0.11.2. Thanks for reporting!
